hub/core/chunk/chunk_compressed_chunk.py

```diff
@@ -297,3 +324,22 @@ def nbytes(self):
     def prepare_for_write(self):
         ffw_chunk(self)
         self.is_dirty = True
+
+    def is_empty_tensor(self):
```

Comment:
or this should be decorated with `@property`?

Reply:
fixed